src/CodeStyle/Core/Analyzers/AnalyzerConfigOptions.cs

```diff
@@ -2,37 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Diagnostics;
 
 namespace Microsoft.CodeAnalysis
 {
-    internal abstract class OptionSet
+    internal sealed class CompilerAnalyzerConfigOptions : AnalyzerConfigOptions
```

Comment:
Is this type being used anywhere now? Seems weird as its TryGetValue always returns false.